src/libcore/sync/atomic.rs

```diff
@@ -1555,7 +1554,7 @@ atomic_int! {
     unstable(feature = "integer_atomics", issue = "32976"),
     unstable(feature = "integer_atomics", issue = "32976"),
     unstable(feature = "integer_atomics", issue = "32976"),
-    unstable(feature = "atomic_nand", issue = "13226"),
+    stable(feature = "atomic_nand", since = "1.27.0"),
```

Comment:
I think for `AtomicU8` etc the method should remain unstable under the feature `integer_atomics`.

Reply:
Thanks, fixed.